tex/chProgramming.tex

```diff
@@ -618,6 +620,11 @@ \section{Data types, first examples}
 sequences of values are among the most prominent examples.
 \index[concept]{inductive type}
 
+In the following sections, we introduce basic data types \C{bool} for boolean
+values and \C{nat} for natural numbers, taking the opportunity to describe
+various constructs of the \Coq{} language as the become relevant.  These
+sections serve simultaneously as an introduction to the data types and to the
```

Comment:
I do not understand "to the data types and to the Coq language constructs". Does this mean something like "to the definition of inductive data types in Coq"?